modules/doc/content/training/index.md

```diff
@@ -1,10 +1,17 @@
 # Training
 
 !style! halign=left
-The following links detail any MOOSE related available training on the horizon.
+Upcoming MOOSE related training:
 !style-end!
 
+- [Programme-MOOSE Modeling and Simulation for Advanced Reactors (2023 Oct 23-26)](https://ncrcaims.inl.gov/Identity/Account/TrainingRegistration)
```

Comment:
what does Programme-MOOSE mean?
is this a NEAMS meeting? NEAMS-training? online?